manifests/public.yaml

```diff
@@ -589,9 +589,6 @@ spec:
   - name: velum-dist-name
     hostPath:
       path: /usr/share/velum/PRODUCT
-  - name: velum-static-pages
-    hostPath:
-      path: /usr/share/velum/public
```

Comment:
we could keep this e.g. with `path: /usr/share/velum/public/static-pages` and then set `paths["public"] = "static-pages"` in https://github.com/kubic-project/velum/blob/master/config/application.rb